src/NuGetTrends.Api/NuGetTrends.Api.csproj

```diff
@@ -11,10 +12,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" />
-    <PackageReference Include="Sentry.AspNetCore" Version="2.0.0-beta2" />
-    <PackageReference Include="Sentry.Serilog" Version="2.0.0-beta2" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc4" />
+    <PackageReference Include="Sentry.AspNetCore" Version="2.0.0-beta6" />
+    <PackageReference Include="Sentry.Serilog" Version="2.0.0-beta6" />
     <PackageReference Include="Serilog.AspNetCore" Version="2.1.1" />
     <PackageReference Include="Serilog.Settings.Configuration" Version="3.1.0" />
```

Comment:
Shouldn't we update these as well?

Reply:
IIRC there were no updates.
I wanted to limit the change also, write some tests after (with C#8 and nullable ref types)
After that we feel more confident with changing stuff